src/transformers/default-schema-transformer.ts

```diff
@@ -259,3 +262,15 @@ function handleQuestionsWithObsComments(sectionQuestions: Array<FormField>): Arr
 
   return augmentedQuestions;
 }
+
+function handleDiagnosis(question: FormField) {
+  if ('dataSource' in question.questionOptions && question.questionOptions['dataSource'] === 'diagnoses') {
+    question.questionOptions.datasource = {
+      name: 'problem_datasource',
+      config: {
+        class: '8d4918b0-c2cc-11de-8d13-0010c6dffd0f',
```

Comment:
Is it possible to provide custom concept classes from the form design perspective?